test/hotspot/jtreg/vmTestbase/nsk/jdi/ThreadReference/interrupt/interrupt001.java

```diff
@@ -211,7 +211,9 @@ private void executeCase(int testCase, String threadName2) {
             throw new TestBug("ERROR: Not found ThreadReference for name :" + threadName2);
         }
 
-        log2("......interrupting the thread2");
+        log2("......thread2 is " + (thread2.isVirtual() ? "" : "not ") + "a virtual thread");
```

Comment:
Nit: Would it better to print "is a virtual thread" instead of "is not a virtual thread"?

Reply:
I don't see your point. It prints one or the other depending on whether or not the thread is virtual.